docs/samples/calling/app.js

```diff
@@ -117,6 +117,15 @@ const getOptionValue = (select) => {
   return selected ? selected.value : undefined;
 };
 
+async function uploadLogs() {
+    try {
+      await callingClient.uploadLogs();
+      console.log('Logs uploaded successfully');
+    } catch (error) {
+      console.error('Failed to upload logs:', error);
+    }
```

Comment:
Can we show these statements in the UI itself? Just in case we end up not uploading logs, the customer would know immediately. 

Also if there's an identifier that can help us filter the logs, we can add that as well in the log.

Something like, "When opening a case with us, please share this tracking ID so that it is easier for us to track and download your logs" - If this is a possibility, it will be great to have that here. Makes it easier for us